source/taup/introduction.rst

```diff
@@ -30,19 +32,23 @@ TauP 目前还存在一些缺点：
 工具
 ----
 
-TauP 提供了多个工具:
+TauP 既提供了图形界面，也提供了命令行工具。
 
 - 图形界面： ``taup``
 - 命令行工具：
 
-  - :doc:`taup_time` ：计算震相的走时、射线参数、出射角、入射角等
-  - :doc:`taup_curve` ：计算走时曲线
-  - :doc:`taup_path` ：计算射线路径
-  - ``taup_pierce`` ：计算震相在波速不连续面和指定深度的反射点和穿透点
-  - ``taup_wavefront`` ：计算波前面
-  - ``taup_table`` ：计算走时表
+  - :doc:`taup_time`\ ：计算震相的走时、射线参数、出射角、入射角等
+  - :doc:`taup_curve`\ ：计算走时曲线
+  - :doc:`taup_path`\ ：计算射线路径
+  - ``taup pierce``\ ：计算震相在波速不连续面和指定深度的反射点和穿透点
+  - ``taup wavefront``\ ：计算波前面
+  - ``taup table``\ ：计算走时表
   - :doc:`taup_setsac` ：将震相走时写入 SAC 头段变量中
-  - ``taup_create`` ：预计算速度模型
+  - ``taup create``\ ：预计算速度模型
+  - ``taup phase``\ ：通过语言描述震相的射线路径
```

Comment:
It seems that `slowness = radius / vel` (s/rad).

Reply:
I don't know. Will explore it when writing the `taup slowplot` tutorial.